src/components/SlateEditor/plugins/uuDisclaimer/utils.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { jsx as slatejsx } from "slate-hyperscript";
+import { TYPE_DISCLAIMER } from "./types";
+import { defaultParagraphBlock } from "../paragraph/utils";
+
+export const defaultDisclaimerBlock = () =>
+  slatejsx(
+    "element",
+    {
+      type: TYPE_DISCLAIMER,
+      data: {
+        resource: "uu-disclaimer",
+        disclaimer: "",
```

Comment:
Putt inn en standardtekst fra phrases-filene her.

Reply:
Enten den eller så kan du sette denne tom for å slippe å innføre et parameter, og heller vise en standardtekst i SlateDisclaimer dersom verdien er tom.